Formula/rex.rb

```diff
@@ -205,10 +227,18 @@ class Rex < Formula
     sha256 "771f7d02abd1ded94d9e37d3f66e795c8d2026d04defbeb5b679ca058116bbf3"
   end
 
+  resource "inc::latest" do
+    url "http://search.cpan.org/CPAN/authors/id/D/DA/DAGOLDEN/inc-latest-0.500.tar.gz"
+    sha256 "daa905f363c6a748deb7c408473870563fcac79b9e3e95b26e130a4a8dc3c611"
+  end
+
   def install
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
     ENV.prepend_path "PERL5LIB", libexec/"lib"
 
+    # Spare Perl v5.23+ users a long-winded gripe from JSON::XS module's author.
```

Comment:
@ilovezfs Did you [see the comment](https://github.com/Homebrew/homebrew-core/pull/14279/files#r123865541) I left on @DomT4's PR? There's nothing to report. The author of the JSON::XS module (who isn't the author of `rex`) is an evangelical idiot with some vendetta against the new Perl 5 maintainers. He's not going to "fix" this because he's **choosing** not to support new Perl versions.